gst/nnstreamer/nnstreamer_subplugin.c

```diff
@@ -67,7 +67,7 @@ static subpluginSearchLogic searchAlgorithm[] = {
   [NNS_SUBPLUGIN_FILTER] = NNS_SEARCH_FILENAME,
   [NNS_SUBPLUGIN_DECODER] = NNS_SEARCH_FILENAME,
   [NNS_EASY_CUSTOM_FILTER] = NNS_SEARCH_FILENAME,
-  [NNS_SUBPLUGIN_CONVERTER] = NNS_SEARCH_GETALL,
+  [NNS_SUBPLUGIN_CONVERTER] = NNS_SEARCH_FILENAME,
```

Comment:
Please state the reason for this.

Reply:
It was changed because `NNS_SEARCH_GETALL` does not seem necessary.
And I'm not aware of what the intention is.

Reply:
The original intention of GETALL is to load all available converter subplugins so that we can automatically choose subplugins based on the incoming type.

E.g., you need GETALL to attach "flatbuf" plugin automatically for ```/other/flatbuf```.